cypress/support/fragments/checkout/checkout.js

```diff
@@ -0,0 +1,18 @@
+import {
+  TextField,
+  Button,
+} from '../../../../interactors';
+
+export default {
+  fillUserBarcode(barcode) {
+    cy.do([
+      TextField({ id: 'input-patron-identifier' }).fillIn(barcode),
+      Button({ id: 'clickable-find-patron' }).click(),
+    ]);
+  },
+  checkUserOpenLoans({ barcode, id }) {
+    this.fillUserBarcode(barcode);
+    cy.wait(5000);
```

Comment:
"for 5000ms before click on user loans in checkout app. There are a lot of requests within user loading, and if leave this page before all of them got responce, test will fail by reason of aborting of requests." - if it's related with this  description - you need to wait expected responses instead

Reply:
I tried what we discussed but it didn't work. I left `cy.wait` here but decreased timer to 3500.